### PR TITLE
Adding code for button import environments

### PR DESCRIPTION
**Pull request by schlupov**
```
def test_positive_import(session):
    name = 'common'
    with session:
        session.puppetenvironment.import_environments(value=name)
        assert session.puppetenvironment.search(
            name)[0]['Name'] == name
```

### CODE REVIEW

File: airgun/entities/puppet_environment.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from airgun.views.puppet_environment import (
     PuppetEnvironmentTableView,
     PuppetEnvironmentCreateView,
+    ImportPuppetEnvironmentView,
```
Comment (abalakh) [inline airgun/entities/puppet_environment.py]: Please sort alphabetically

File: airgun/entities/puppet_environment.py [hunk 2]
```diff
@@ -34,6 +35,11 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def import_environments(self, value):
```
Comment (abalakh) [inline airgun/entities/puppet_environment.py]: Please add docstring

File: airgun/entities/puppet_environment.py [hunk 3]
```diff
@@ -67,3 +73,14 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+
+@navigator.register(PuppetEnvironmentEntity, 'Import')
+class ImportPuppetEnvironmentView(NavigateStep):
```
Comment (abalakh) [inline airgun/entities/puppet_environment.py]: >Please add docstring

File: airgun/views/puppet_environment.py [hunk 4]
```diff
@@ -55,6 +55,14 @@ class ImportPuppetEnvironmentView(BaseLoggedInView, SearchableViewMixin):
     obsolete = Text("//a[contains(@data-original-title,'obsolete')]")
     update = Text("//input[@name='commit']")
     cancel = Text("//a[@data-id='aid_environments']")
+    table = SatTable(
+        locator='.//table',
+        column_widgets={
+            'Environment': Text('//a[starts-with(\
```
Comment (abalakh) [inline airgun/views/puppet_environment.py]: How many links are there in 'Environment' column? If that's the only one - just use `./a` instead. And btw use relative path (starting with `.`).

**Comment by oshtaier** [comment]
Looks good.
So what is a result for `test_positive_import`?

**Comment by schlupov** [comment]
The test passes

screenshot from 2018-07-30 09-06-20

**Pull request merged**